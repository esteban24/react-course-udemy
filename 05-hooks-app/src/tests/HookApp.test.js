import React from 'react';
import { shallow } from 'enzyme';
import { HookApp } from '../HookApp';

describe('HookApp.js test suite', () => {
    test('should match snapshot', () => {
        const wrapper = shallow(<HookApp/>);

        expect(wrapper).toMatchSnapshot();
    })
})
