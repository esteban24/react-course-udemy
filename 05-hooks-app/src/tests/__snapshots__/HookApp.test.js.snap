// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookApp.js test suite should match snapshot 1`] = `
<h1>
  Hello World!
</h1>
`;
